sdk/src/transaction.rs

```diff
@@ -374,14 +351,15 @@ mod tests {
         let expected_transaction_size = 1
             + (tx.signatures.len() * size_of::<Signature>())
             + 1
-            + (tx.account_keys.len() * size_of::<Pubkey>())
+            + 1
```

Comment:
Same question as Rob's about this magic number. I assume it is the num_signatures length, but some comments on these integers might make it more clear, and make upkeep easier.

Reply:
done